### PR TITLE
Fee Rate Tuning

### PR DESCRIPTION
**Pull request by dannydeezy**
In low-fee environments, we've noticed the hunter more frequently misses confirming in the next block. With this change here, we will be more aggressive in low fee environments without changing the behavior of high fee environments too much

### CODE REVIEW

**Comment by dannydeezy** [comment]
In a high fee environment, the min next block fee might be something like `46.2`, in which case we'd now use `47.0` and it wouldn't change much.

In a low fee environment, the min next block fee is more likely to be something like `8.2`, in which case we'd use `9.0`. This can easily make the difference between getting into the next block or not

File: index.js [hunk 1]
```diff
@@ -181,7 +181,7 @@ async function run() {
         if (input_utxo) {
             console.log(`Found existing unconfirmed send with fee rate of ${existing_fee_rate} sat/vbyte`)
             console.log(`Current fee rate is ${fee_rate} sat/vbyte`)
-            if (fee_rate - existing_fee_rate >= 2) {
+            if (fee_rate - existing_fee_rate >= 1.2) {
```
Comment (HashTraveler) [inline index.js:184]: I would add 1.2 as a const with meaningful name, ex. maybe: 
```
const MINIMUM_EXISITNG_FEE_THRESHOLD = 1.2
```

File: fees.js [hunk 2]
```diff
@@ -41,10 +42,10 @@ async function get_min_next_block_fee_rate() {
     }
     if (MIN_FEE_BUFFER_PERCENT) {
         const min_fee_buffer_percent = parseFloat(process.env.MIN_FEE_BUFFER_PERCENT)
-        return Math.round((data[0].feeRange[0] * min_fee_buffer_percent) * 10) / 10
+        return Math.ceil((data[0].feeRange[NEXT_BLOCK_FEE_SLOT] * min_fee_buffer_percent) * 10) / 10
```
Comment (HashTraveler) [inline fees.js:45]: Shall we cover this method `get_fee_rate` with unit tests?
changing round to ceil should cause unit test failure as this is a change in logic

**Review by topether21 — APPROVED** [review]
I agree with @HashTraveler about `const MINIMUM_EXISITNG_FEE_THRESHOLD = 1.2`

**Review by HashTraveler — APPROVED** [review]

**Review by habibitcoin — APPROVED** [review]

**Pull request merged**